modules/hrm/includes/functions-employee.php

```diff
@@ -323,8 +323,16 @@ function erp_employee_delete( $employee_ids, $force = false ) {
         if ( $force ) {
 
             // find leave entitlements and leave requests and delete them as well
-            \WeDevs\ERP\HRM\Models\Leave_Request::where( 'user_id', '=', $employee_wp_user_id )->delete();
+            $leave_requests = \WeDevs\ERP\HRM\Models\Leave_Request::where( 'user_id', '=', $employee_wp_user_id )->get()->toArray();
```

Comment:
No need to use ->toArray() function... you can access id inside loop with $lr->id